crux-core/src/crux/node.clj

```diff
@@ -194,9 +192,9 @@
                                :closed? (atom false)
                                :lock (StampedLock.)}))
    :deps #{::indexer ::kv-store ::bus ::object-store ::tx-log}
-   :args {:crux.tx-log/await-tx-timeout {:doc "Default timeout in milliseconds for waiting."
-                                         :default 10000
-                                         :crux.config/type :crux.config/nat-int}}})
+   :args {:crux.tx-log/await-tx-timeout {:doc "Default timeout for awaiting transactions being indexed."
+                                         :default nil
```

Comment:
think you can just remove default then?

Reply:
yep - it's `crux.topology/parse-opts` that'd would complain - it checks whether the value's required, and then only runs the validate-fn if the value's non nil.

`(parse-opts {:crux.tx-log/await-tx-timeout {:default nil, :crux.config/type :crux.config/duration}} {})`
=> `{:crux.tx-log/await-tx-timeout nil}`